pkg/app/grpCurl.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+type grpcurl struct {
+	ns         string
+	securePort string
+	secureHost string
+}
+
+var _ App = &grpcurl{}
+
+func GrpCurl(ns string, securePort string, secureHost string) App {
+
+	grpCurl := grpcurl{
+		ns:         ns,
+		securePort: securePort,
+		secureHost: secureHost,
+	}
+
+	return &grpCurl
+
```

Comment:
This function could be simpler:
```suggestion
	return &grpcurl{
		ns:         ns,
		securePort: securePort,
		secureHost: secureHost,
	}
```

Reply:
Done. ty